styles.css

```diff
@@ -590,6 +711,126 @@ section.footer-m {
   background-size: cover;
 }
 
+/* Pop-up */
+
+section#pop-up {
+  width: 100%;
+  height: 100%;
+  background: #1a2236;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 2;
+  box-sizing: border-box;
+}
+
+.pop-up-content {
+  display: flex;
+  flex-direction: column;
+  background: white;
+  height: 100%;
+  padding: 20px 20px;
+  box-sizing: border-box;
+  overflow-y: scroll;
+}
+
+.pop-up-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+}
+
+.pop-up-header .menu-btn {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.image-gallery-container {
+  display: flex;
+  flex-direction: column;
+  margin-top: 30px;
+}
+
+.selected-image-container {
+  position: relative;
+  width: 100%;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.gallery {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  justify-content: flex-start;
+  margin: 0;
+}
+
+.gallery-item-container {
+  display: flex;
+  width: 25%;
+  height: auto;
+}
+
+.gallery-image {
+  width: 100%;
+  height: auto;
+  object-fit: contain;
+}
+
+.gallery-control {
+  position: absolute;
+  border: none;
+  display: flex;
+  height: 99%;
+  justify-content: center;
+  align-items: center;
+}
+
+img.gallery-icon {
+  width: 10px;
+}
+
+.prev {
+  left: 0;
+  background: linear-gradient(90deg, #0000007a, transparent);
+}
+
+.next {
+  right: 0;
+  background: linear-gradient(270deg, #0000007a, transparent);
+}
```

Comment:
In order to match the design, kindly remove the prev and next buttons from the mobile version since it is not suppose to be there. :wink: 

### Your design 

![Screenshot from 2022-12-21 22-58-23](https://user-images.githubusercontent.com/65233860/209019308-d0b40461-38aa-4603-9029-ad1b8dd727c1.png)



### Figma design 

![Screenshot from 2022-12-21 22-58-12](https://user-images.githubusercontent.com/65233860/209019329-1adbdf94-d375-430e-88f4-0044ef37ccd5.png)

Reply:
The design with is 320px, and you are checking the project in 767px.